openbas-front/src/admin/components/assets/endpoints/Endpoints.tsx

```diff
@@ -158,7 +159,7 @@ const Endpoints = () => {
           <ListItemSecondaryAction />
         </ListItem>
         {endpoints.map((endpoint: Endpoint) => {
-          const executor = executorsMap[endpoint.asset_executor ?? 'Unknown'];
+          const executor = executorsMap[endpoint.asset_agents && endpoint.asset_agents[0].agent_executor ? endpoint.asset_agents[0].agent_executor : 'Unknown'];
```

Comment:
**thought:**

Should be better to have a new property on back-end.
What do you think ?

  /** Used to show Front column */
  @JsonProperty("asset_executor")
  public boolean getExecutor() {
    return this.agents.getFirst().getExecutor();
  }

Same thinking for other properties that are searched via getFirst()

Reply:
I prefer doing this only for properties used for the future task "rework endpoints view". We will just have to replace the inside of each getter to send what the new endpoints view will need.
For the other agent attributes, I prefer keeping the "getAgents().getFirst()" in the code to replace in the future by "forEach" methods.
Is it OK for you ? Thanks !